cmd/crc/cmd/delete_test.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/code-ready/crc/pkg/crc/machine/fakemachine"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPlainDelete(t *testing.T) {
+	cacheDir, err := ioutil.TempDir("", "cache")
+	require.NoError(t, err)
+	defer os.RemoveAll(cacheDir)
+
+	out := new(bytes.Buffer)
+	assert.NoError(t, runDelete(out, fakemachine.NewClient(), true, cacheDir, true, true, ""))
+	assert.Equal(t, "Deleted the OpenShift cluster\n", out.String())
+
+	_, err = os.Stat(cacheDir)
+	assert.True(t, os.IsNotExist(err))
+}
+
+func TestJSONDelete(t *testing.T) {
+	cacheDir, err := ioutil.TempDir("", "cache")
+	require.NoError(t, err)
+	defer os.RemoveAll(cacheDir)
+
+	out := new(bytes.Buffer)
+	assert.NoError(t, runDelete(out, fakemachine.NewClient(), true, cacheDir, false, true, jsonFormat))
+	assert.Equal(t, `{
+  "success": true
+}
+`, out.String())
+
```

Comment:
Did you try `assert.JSONEq(t, ``{"success": true}``, out.String())` to avoid the slightly hard to parse/read multiline string?

Reply:
I didn't know this assertions, thanks! I will use it now.